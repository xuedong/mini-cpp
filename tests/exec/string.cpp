#include <iostream>

int main() {
  std::cout << "abc\n";
  std::cout << "a\nc\n";
  std::cout << "a\\c\n";
  std::cout << "a\tc\n";
  std::cout << "a\"c\n";
  std::cout << "\x41\x42\x43\n";
}
