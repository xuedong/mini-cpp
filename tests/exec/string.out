abc
a
c
a\c
a	c
a"c
ABC
